app/views/membership_applications/_application_status_form.html.haml

```diff
@@ -1,9 +1,15 @@
-= form_for @membership_application, url: membership_application_path do |f|
-  %h3= t('membership_applications.update_the_status')
-  = f.select(:status, [["#{t('membership_applications.pending')}", 'Pending'],
-                       ["#{t('membership_applications.treated')}", 'Behandlas'],
-                       ["#{t('membership_applications.awaiting_payment')}", 'Inväntar betalning'],
-                       ["#{t('membership_applications.pending_completion')}", 'Inväntar komplettering'],
-                       ["#{t('membership_applications.accepted')}", 'Godkänd'],
-                       ["#{t('membership_applications.rejected')}", 'Avböjd']])
-  = f.submit "#{t('update')}", class: 'wpcf7-submit file'
+#admin-change-status-buttons
+  %p.header
+    = "#{t('membership_applications.show.change_status')}:"
+
+  = button_to t('membership_applications.accept'), accept_membership_application_path(@membership_application), {class: 'btn accept', disabled: !(@membership_application.may_accept?)}
+
+  = button_to t('membership_applications.reject'), reject_membership_application_path(@membership_application), {class: 'btn reject', disabled: !(@membership_application.may_reject?)}
+
+  = button_to t('membership_applications.ask_applicant_for_info'), need_info_membership_application_path(@membership_application), {class: 'btn need-info', disabled: !(@membership_application.may_ask_applicant_for_info?)}
+
+  = button_to t('membership_applications.cancel_waiting_for_applicant'), cancel_need_info_membership_application_path(@membership_application), {class: 'btn cancel-need-info', disabled: !(@membership_application.may_cancel_waiting_for_applicant?)}
```

Comment:
I need to update the diagram.  Also, we haven't decided on the states and transitions yet.  I started with these, but need to change them according to decisions/feedback.